spec/request/space_manifests_spec.rb

```diff
@@ -1029,9 +1029,10 @@
 
         it 'returns an appropriate error' do
           headers = yml_headers(user_header)
-          headers['CONTENT_TYPE'] = 'bogus'
+          headers['CONTENT_TYPE'] = 'application/bogus'
```

Comment:
this test started returning [406](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406) when `CONTENT_TYPE` was `bogus`.   Seems reasonable to me.